linkerd/app/outbound/src/http/logical/policy/route.rs

```diff
@@ -109,6 +110,8 @@ where
                 // consideration, so we must eagerly fail requests to prevent
                 // leaking tasks onto the runtime.
                 .push_on_service(svc::LoadShed::layer())
+                // Sets an optional request timeout.
+                .push(http::NewTimeout::layer())
```

Comment:
genuine question, not really feedback: why not apply the timeout before the filters? for example why not wrap a redirect filter in the timeout? wouldn't it fall under the work done in the proxy?

Reply:
No good reason, I wasn't thinking too hard about this. It could probably be moved to before filters.

Reply:
moved in https://github.com/linkerd/linkerd2-proxy/pull/2418/commits/38f3f2beda0fa557a6092c9014d62c353480e636